src/ngamsPlugIns/ngamsPlugIns/ngamsCmd_MIRREXEC.py

```diff
@@ -779,7 +779,7 @@ def send_mirrexec_command(self):
             start_time = time.time()
             response = ngamsHttpUtils.httpGet(host, int(port), 'MIRREXEC', pars=pars, timeout=self.rx_timeout)
             with contextlib.closing(response):
-                failed = response.status != NGAMS_HTTP_SUCCESS or NGAMS_FAILURE in response.read()
+                failed = response.status != NGAMS_HTTP_SUCCESS or NGAMS_FAILURE in str(response.read)
```

Comment:
Maybe you meant `str(response.read())`? Otherwise you're making a string out of the `read` method itself.

Anyways, `str(response.read())` won't work either, because you can't just create a `str` out of a `bytes` object. What you want is to use:

```python
from ngamsLib.utils import b2s

if NGAMS_FAILURE in b2s(response.read()):
   pass
```

Reply:
Thank you for correcting me. I have committed the solution you suggested.